fondant/cli.py

```diff
@@ -0,0 +1,71 @@
+"""This file contains CLI script for the fondant package.
+
+To add a script, add a function to this file and add it to `pyproject.toml` file
+under the `[tool.poetry.scripts]` section.
+To add a script, use the following format:
+
+    [tool.poetry.scripts]
+    script_name = "fondant.cli:script_function"
+
+When installing the fondant package, the script will be available in the
+environment.
+
+e.g.
+
+    'fondant-explorer --source /path/to/data'
+    'script_name --arg1 --arg2''
+"""
+import argparse
+import logging
+import shlex
+import shutil
+import subprocess  # nosec
+
+DEFAULT_REGISTRY = "ghcr.io/ml6team/data_explorer"
+DEFAULT_TAG = "latest"
+DEFAULT_PORT = "8501"
+
+
+def run_data_explorer():
+    """Run the data explorer container."""
+    logging.basicConfig(level=logging.INFO)
+
+    parser = argparse.ArgumentParser(description="Run the data explorer container.")
+    parser.add_argument("--source", "-s", help="Path to the source directory.")
```

Comment:
It's not clear to me what is expected as the source directory here. Can we add a better description (and maybe better name?)

And can we make this work with relative paths as well?

Reply:
It's a local folder that contains the data (manifest etc), which you mount into the Docker container. I'll think of a better name

Reply:
`--data-directory` is a lot more clear to me :) 

Would be ideal if we can develop a common language instead of referring to the same thing by different names. So would be great if we can align this with fondant. Maybe we should rename `base_path` do `data_directory` (or something similar) there as well @GeorgesLorre @PhilippeMoussalli?